src/gst/fluidsynth/fluidsynth.c

```diff
@@ -587,7 +587,7 @@ gstbt_fluid_synth_get_property (GObject * object, guint prop_id,
           g_value_set_double (value, d);
         break;
       case G_TYPE_STRING:
-        retval = fluid_settings_getstr (src->settings, name, &s);
+        retval = fluid_settings_dupstr (src->settings, name, &s);
```

Comment:
Please move fluidsynth compatibility changes to a new PR. Thanks for looking into those. I think we can quickly make this mergable.

Reply:
Also fluid_settings_dupstr() will return a copy, which we'd need to free, but we can also pass ownership, by changing the g_value_set_string() below into g_value_take_string().

Reply:
I'll move into a new PR and fix it up. To be honest, I haven't even tested this so I'll make sure to do that this week too. All I know so far is that it builds. Sorry for the messiness here.